packages/agent-kit/src/boards/specialist.ts

```diff
@@ -74,8 +79,45 @@ You are very creative and you pride yourself in adding interesting twists and un
 const sampleTask: LlmContent = contextFromText(`
 Write an outline for a novel, following the provided specs.
 `);
+const context = input({
+  title: "Context in",
+  description: "Incoming conversation context",
+  type: annotate(array("string"), {
+    behavior: ["llm-content"],
+  }),
+  examples: [[JSON.stringify(sampleContext, null, 2)]]
```

Comment:
You shouldn't ever need the JSON serialize examples with the build API. It expects the actual value type.

Also here and below, there seem to be two layers of arrays? Like this should be an array of strings, but it's actually an array *of arrays* of strings.